pyo3-build-config/src/impl_.rs

```diff
@@ -456,6 +485,37 @@ print("mingw", get_platform().startswith("mingw"))
         }
         Ok(())
     }
+
+    /// Run a python script using the executable of this InterpreterConfig with additional
+    /// environment variables (e.g. PYTHONPATH) set.
+    pub fn run_python_script_with_envs<I, K, V>(&self, script: &str, envs: I) -> Result<String>
+    where
+        I: IntoIterator<Item = (K, V)>,
+        K: AsRef<OsStr>,
+        V: AsRef<OsStr>,
+    {
+        run_python_script_with_envs(
+            Path::new(
+                self.executable
+                    .as_ref()
+                    .expect("No interpreter executable!"),
+            ),
+            script,
+            envs,
+        )
+    }
+
+    /// Run a python script using the executable of this InterpreterConfig.
+    pub fn run_python_script(&self, script: &str) -> Result<String> {
+        run_python_script(
```

Comment:
Might be worth making this call `self.run_python_script_with_envs`, for simplicity?